platform/ios/src/MGLFaux3DUserLocationAnnotationView.mm

```diff
@@ -1,6 +1,7 @@
 #import "MGLFaux3DUserLocationAnnotationView.h"
 
 #import "MGLMapView.h"
```

Comment:
Please remove this import.

Reply:
we moved one function to map view private which is why we included it.

Reply:
Right. That's why you don't have to import `MGLMapView.h` when you have `MGLMapView_Private.h`

Reply:
gotcha, I can remove it